index.html

```diff
@@ -195,6 +195,30 @@ <h2>About
         </ul>
       </div>
     </div>
+    <div class="contact-me">
```

Comment:
- [ ] kindly consider sticking to the design templates in Figma as much as possible, this includes the font, colors, images, and spacing (margin/padding) to make your design an exact duplicate of the given ones  check the images below for more clarification 
- your design 👇 
<img width="287" alt="Screen Shot 2023-06-13 at 16 52 56" src="https://github.com/Edgahkipkemoi/mobile/assets/60146030/8e6a9b7d-5892-46da-8a7b-a50424722b8d">

- Figma design 👇 
<img width="295" alt="Screen Shot 2023-06-13 at 16 56 47" src="https://github.com/Edgahkipkemoi/mobile/assets/60146030/ee701898-6dac-4711-93ee-825a41c4705b">

Reply:
I have made the all the changes can you please help me identify the background image because on my end I have the background image.